README.md

```diff
@@ -10,8 +10,9 @@ circe-generic in that this derivation is not based on Shapeless's generic repres
 classes and ADTs, and it is not intended to be as fully featured as `io.circe.generic`. In
 particular:
 
-* It's not published for Scala 2.10.
+* Sealed hierarchies (ADTs) aren't supported
```

Comment:
For what it's worth this is coming soon (but I agree it's good to note it for now).